component/avatar/avatar.html

```diff
@@ -0,0 +1,184 @@
+                    style="width: 955px; height: 336px; border:0; transform: scale(1); overflow:hidden;"
+                    sandbox="allow-scripts allow-same-origin">
+                </iframe>
+            </div>
+
+            <h2>Rounded avatar in different sizes</h2>
+            <p>It can be used to show avatar of a profile in round shape. It can be used in three sizes: <span>Large |
+                    Medium | Small</span></p>
+
+            <div class="preview-and-code">
+                <div class="avatar-container">
+
+                    <div class="avatar">
+                        <img src="..\..\assets\images\profile-picture.jpg" alt="large size avatar image"
+                            class="round-avatar-image large-square-size">
+                    </div>
+
+
+                    <div class="avatar">
+                        <img src="..\..\assets\images\profile-picture.jpg" alt="medium size avatar image"
+                            class=" round-avatar-image medium-square-size">
+                    </div>
+
+                    <div class="avatar">
+                        <img src="..\..\assets\images\profile-picture.jpg" alt="small size avatar image"
+                            class=" round-avatar-image small-square-size">
+                    </div>
+                </div>
+                <iframe
+                    src="https://carbon.now.sh/embed?bg=rgba%280%2C0%2C0%2C1%29&t=base16-dark&wt=none&l=htmlmixed&width=955&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522avatar%2522%253E%250A%2520%2520%253Cimg%2520src%253D%2522..%255C..%255Cassets%255Cimages%255Cprofile-picture.jpg%2522%2520alt%253D%2522large%2520size%2520avatar%2520image%2522%2520class%253D%2522round-avatar-image%2520large-square-size%2522%253E%250A%253C%252Fdiv%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%250A%2520%2520%2520%2520%2520%2520%2520%2520%250A%253Cdiv%2520class%253D%2522avatar%2522%253E%250A%2520%2520%253Cimg%2520src%253D%2522..%255C..%255Cassets%255Cimages%255Cprofile-picture.jpg%2522%2520alt%253D%2522medium%2520size%2520avatar%2520image%2522%2520class%253D%2522%2520round-avatar-image%2520medium-square-size%2522%253E%250A%253C%252Fdiv%253E%250A%250A%253Cdiv%2520class%253D%2522avatar%2522%253E%250A%2520%2520%2520%253Cimg%2520src%253D%2522..%255C..%255Cassets%255Cimages%255Cprofile-picture.jpg%2522%2520alt%253D%2522small%2520size%2520avatar%2520image%2522%2520class%253D%2522%2520round-avatar-image%2520small-square-size%2522%253E%250A%253C%252Fdiv%253E"
+                    style="width: 955px; height: 336px; border:0; transform: scale(1); overflow:hidden;"
+                    sandbox="allow-scripts allow-same-origin">
+                </iframe>
```

Comment:
instead of using carbon I suggest you use prims js and that way your website will be responsive as well.